strapi/strapi
# Fix Factories typings

### What does it do?

Fix the Factories Typings

### Why is it needed?

To provide a good, and correct linting experience when using `createCoreController` or `createCoreService` or `createCoreRouter`

### How to test it?

Try to add or wrap services or controllers or routes, it should work as explained in the docs:
https://docs.strapi.io/developer-docs/latest/development/backend-customization/routes.html#implementation
https://docs.strapi.io/developer-docs/latest/development/backend-customization/controllers.html#implementation
https://docs.strapi.io/developer-docs/latest/development/backend-customization/services.html

### Related issue(s)/PR(s)

#13241
